projects/week_1/Instructions.md

```diff
@@ -0,0 +1,332 @@
+
+The last thing we need to do for this section is to copy the new position's coordinates into the `new_tag` message. Copying the x value looks like this:
+
+```c++
+new_tag.pose.position.x = position.x();
+```
+
+Do the same for y and z.
+
+### 2.8 Transform tag orientation
+
+To transform the tag's orientation, we're going to follow a similar process. We'll start by extracting the rotation from the old tag message. We've provided a function to help you with this. Declare a variable named `tag_orientation` of type `Eigen::Matrix4d` right after you position code. Initialize `tag_orientation` by calling `quaternionMessageToTransformationMatrix()`, passing it `tag_array_msg->tags[i].pose.orientation`.
+
+Now, apply both transformations to `tag_orientation`. This should look basically identical to how you applied the transformations to `position`.
+
+Finally, we need to convert our new orientation back into a quaternion message and set it in the `new_tag` message. Here again, we've got a helper function for you called `transformationMatrixToQuaternionMessage()`. Call this function, passing `tag_orientation`, and assign the result to `new_tag.pose.orientation`.
+
+And that's it! We've written all the code we need to make this node complete.
+
+### 2.9 Build project
+
+Before we can run our newly finished node, we need to build it. Open a terminal window and navigate to the training workspace folder (`/home/robojackets/training_ws`).
+
+Source the underlay setup script:
+
+```bash
+$ source /opt/ros/foxy/setup.bash
+```
+
+And use colcon to build the workspace
+
+```bash
+$ colcon build
+```
+
+If your build succeeded, the last line printed will be `Summary: __ packages finished [_.__s]` that tells you how many packages were built and how long the process took.
+
+If any errors were found during the build, the final summary will tell you, and the output will include the error details. The summary of a failed build looks like this:
+
+```bash
+Summary: 16 packages finished [5.12s]
+  1 package failed: coordinate_transform
+  1 package had stderr output: coordinate_transform
+```
+
+If you do have any errors, you'll need to fix them before you can run your node. We recommend starting with the first error reported, reading the error message carefully, fixing it, and building again. Repeat that process until all errors are resolved. Of course, if you're not sure how to fix any given error, reach out to the trainers.
+
+### 2.10 Run project
+
+To run this project, we'll need two terminal windows or tabs.
+
+In the first terminal, after sourcing the underlay setup file and the training workspace overlay setup file, launch the week 1 launch file.
```

Comment:
I would split the sourcing overlay and underlay into a separate box. I don't want them to start to think those three commands equal a launch, which is what I would think reading this.